lib/tasks/data_demo.rake

```diff
@@ -5,6 +5,35 @@ namespace :data do
     conference.contact.update_attributes(email: 'osemdemo@osem.io', sponsor_email: 'osemdemo@osem.io')
-    create(:admin, email: 'admin@osem.io', username: 'admin', password: 'password123', password_confirmation: 'password123')
+    create(:conference, title: 'Demo Conference', short_title: 'Opensuse' ,description: "This is a demo conference instance. No information about this conference is set by default.")
+
+    program = conference.program
+
+    admin = create(:admin, email: 'admin@osem.io', username: 'admin', password: 'password123', password_confirmation: 'password123')
+    user1 = create(:user, email: 'user1@osem.io', username: 'user1', password: 'password123', password_confirmation: 'password123')
+    user2 = create(:user, email: 'user2@osem.io', username: 'user2', password: 'password123', password_confirmation: 'password123')
+
+    selected_schedule = create(:schedule, program: program)
+    demo_schedule = create(:schedule, program: program)
+    program.update_attributes(selected_schedule: selected_schedule)
+    
+    create(:event, program: program, title: 'Demo Event', abstract: 'This is a demo event instance with its state not defined')
+    create(:event, program: program, title: 'Demo Rejected Event', state: 'rejected', abstract: 'This is demo event instance in a rejected state')
+    create(:event, program: program, title: 'Demo Unconfirmed Event', state: 'unconfirmed', abstract: 'This is a demo event instance in unconfirmed state')
+    create(:event, program: program, title: 'Demo Confirmed Event', state: 'confirmed', abstract: 'This is a demo event instance in a confirmed state')
+
+    first_scheduled_event = create(:event, program: program, title: 'first_scheduled_event', state: 'confirmed', abstract: 'This is a demo scheduled event instance')
+    second_scheduled_event = create(:event, program: program, title: 'second_scheduled_event', state: 'confirmed', abstract: 'This is a demo scheduled event instance')
+    multiple_speaker_event = create(:event, program: program, title: 'multiple_speaker_event', state: 'confirmed', abstract: 'This is a [Open Source Event demo scheduled event instance having multiple speakers')
+
+    create(:event_user, event: multiple_speaker_event, user: user1, event_role: "speaker")
+    create(:event_user, event: multiple_speaker_event, user: user2, event_role: "speaker")
+
+    create(:event_schedule, event: first_scheduled_event, schedule: selected_schedule, start_time: conference.start_date.to_time + 20.hours, room: Room.first)
+    create(:event_schedule, event: second_scheduled_event, schedule: selected_schedule, start_time: conference.start_date.to_time + 20.hours, room: Room.second)
```

Comment:
It would be better if not all the events start at the same time. For example:

``` Ruby
create(:event_schedule, event: second_scheduled_event, schedule: selected_schedule, start_time: conference.start_date.to_time + 20.hours + 30.minutes, room: Room.second)
create(:event_schedule, event: multiple_speaker_event, schedule: selected_schedule, start_time: conference.start_date.to_time + 20.hours + 15 minutes, room: Room.third)
```